packages/design-system/src/styles/components/_Tabs.scss

```diff
@@ -16,7 +16,7 @@
   padding: $spacer-3;
 
   &[aria-hidden='true'] {
-    display: none !important;
+    display: none;
```

Comment:
Tested locally and the `!important` didn't seem necessary.

TODO: Perhaps Tabs should be refactored to use `hidden` instead of `aria-hidden`? The `hidden` attr has the same effect as `aria-hidden` in addition to visually hiding the content when applied - which is what's being done with this CSS.

Reply:
Maybe it would be worth adding a "TODO" comment in the code 😆